themes/default/content/blog/pulumi-release-notes-77/index.md

```diff
@@ -27,6 +27,7 @@ authors:
 # At least one tag is required. Lowercase, hyphen-delimited is recommended.
 tags:
    - features
+   - pulumi-releases
```

Comment:
Should this be `releases` so it starts to fill out this tag @SaraDPH? https://www.pulumi.com/blog/tag/releases/

Or would you like to replace `releases` with `pulumi-releases` and have everything go into that?

Reply:
Pulumi-releases, that's what was agreed with others too and be top priority in the cloud :)